.circleci/config.yml

```diff
@@ -140,9 +156,7 @@ jobs:
           key: v{{ checksum "cache.version" }}-yarn-vendor-{{ checksum "yarn.lock" }}
           paths:
             - /usr/local/share/.cache/yarn
-      - run: yarn workspace @chainlink/json-api-client
-      - store_artifacts:
-          path: ./integration/logs
+      - run: yarn workspace @chainlink/json-api-client run test
```

Comment:
Tip: you can omit `run` and it'll still work :D

Reply:
👍 because it's CI config I prefer it to be the explicit full command

Reply:
sounds good